akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -492,6 +492,19 @@ trait FlowOps[+Out, +Mat] {
   def filter(p: Out ⇒ Boolean): Repr[Out, Mat] = andThen(Filter(p.asInstanceOf[Any ⇒ Boolean]))
 
   /**
+   * Only pass on those elements that NOT satisfy the given predicate.
+   *
+   * '''Emits when''' the given predicate returns false for the element
+   *
+   * '''Backpressures when''' the given predicate returns false for the element and downstream backpressures
+   *
+   * '''Completes when''' upstream completes
+   *
+   * '''Cancels when''' downstream cancels
+   */
+  def filterNot(p: Out ⇒ Boolean): Repr[Out, Mat] = filter(!p(_))
```

Comment:
`via(Flow[Out].filter(!p(_)).withAttributes(...))` will fix it without the need for a new stage